### PR TITLE
Sustained immunity update

### PR DESCRIPTION
**Pull request by RLordan**
Hey @rando2! The sustained immunity section has been updated in light of recent studies showing sustained immunity. I’d appreciate some suggestions for reviewers and edits. 
Thanks a million!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
 They reported that they could detect IgG within two days of the onset of fever and were not able to detect IgM any earlier, a pattern they compared to findings in MERS.
+Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].
+
 <!--TODO: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->
-The duration over which these antibodies persist remains unknown.
+Evidence to date suggests that sustained immunity to SARS-CoV-2 occurs for a period of at least 6-8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @doi:10.1126/sciimmunol.abf8891].
+One study assessed sustained immunity using blood samples from 188 COVID-19 positive patients at various time points between 6 and 240 days post-symptom onset, whereby some patients were also assessed longitudinally to determine [@doi:10.1126/science.abf4063].
```
Comment (cgreene) [inline content/10.diagnostics.md:190]: does this sentence end early?

Comment (cgreene) [reply content/10.diagnostics.md:190]: or late!
```suggestion
One study assessed sustained immunity using blood samples from 188 COVID-19 positive patients at various time points between 6 and 240 days post-symptom onset, whereby some patients were also assessed longitudinally [@doi:10.1126/science.abf4063]. 
```

File: content/10.diagnostics.md [hunk 2]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
+
 <!--TODO: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->
-The duration over which these antibodies persist remains unknown.
+Evidence to date suggests that sustained immunity to SARS-CoV-2 occurs for a period of at least 6-8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @doi:10.1126/sciimmunol.abf8891].
+One study assessed sustained immunity using blood samples from 188 COVID-19 positive patients at various time points between 6 and 240 days post-symptom onset, whereby some patients were also assessed longitudinally to determine [@doi:10.1126/science.abf4063]. 
+After 1 month, 98% of patients were seropositive for spike IgG. 
+Moreover, spike IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, whereby 90% of subjects were seropositive at 6 months.
+Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-C0V-2 neutralizing antibodies.
```
Comment (cgreene) [inline content/10.diagnostics.md:193]:
```suggestion
Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.
```

**Review by cgreene — COMMENTED** [review]
Just a few suggestions!

**Comment by marouenbg** [comment]
Hi @RLordan , this is pretty cool :smiley:, I added the sustained immunity papers last night to update this section but I wouldn't have done a better writing than you did. I might do a PR to talk about the discrepancies in the results between the studies done in UK/USA vs Brazil/Kenya/India and link to social determinants of SARS-CoV-2.

File: content/10.diagnostics.md (hunk 1 above)
Comment (RLordan) [reply content/10.diagnostics.md:190]: My bad! Must have missed it when editing

**Comment by RLordan** [comment]
> Hi @RLordan , this is pretty cool 😃, I added the sustained immunity papers last night to update this section but I wouldn't have done a better writing than you did. I might do a PR to talk about the discrepancies in the results between the studies done in UK/USA vs Brazil/Kenya/India and link to social determinants of SARS-CoV-2.

Hey @marouenbg , I look forward to seeing your PR, that would be very interesting to do! Thanks a million, feel free to edit etc. :)

**Comment by RLordan** [comment]
Thanks @cgreene for the edits!

File: content/10.diagnostics.md [hunk 3]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
 They reported that they could detect IgG within two days of the onset of fever and were not able to detect IgM any earlier, a pattern they compared to findings in MERS.
+Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].
+
 <!--TODO: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->
-The duration over which these antibodies persist remains unknown.
+Evidence to date suggests that sustained immunity to SARS-CoV-2 occurs for a period of at least 6-8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @doi:10.1126/sciimmunol.abf8891].
+One study assessed sustained immunity using blood samples from 188 COVID-19 positive patients at various time points between 6 and 240 days post-symptom onset, whereby some patients were also assessed longitudinally [@doi:10.1126/science.abf4063].
```
Comment (rando2) [inline content/10.diagnostics.md:190]:
```suggestion
One study assessed sustained immunity using 245 blood samples from 188 COVID-19 positive patients  [@doi:10.1126/science.abf4063]. 
The samples were collected at various time points between 6 and 240 days post-symptom onset, meaning some patients were assessed longitudinally. 
Of the samples, 43 were collected at least 6 months after symptom onset.
```

File: content/10.diagnostics.md [hunk 4]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
 They reported that they could detect IgG within two days of the onset of fever and were not able to detect IgM any earlier, a pattern they compared to findings in MERS.
+Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].
+
 <!--TODO: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->
-The duration over which these antibodies persist remains unknown.
+Evidence to date suggests that sustained immunity to SARS-CoV-2 occurs for a period of at least 6-8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @doi:10.1126/sciimmunol.abf8891].
+One study assessed sustained immunity using blood samples from 188 COVID-19 positive patients at various time points between 6 and 240 days post-symptom onset, whereby some patients were also assessed longitudinally [@doi:10.1126/science.abf4063]. 
+After 1 month, 98% of patients were seropositive for spike IgG.
```
Comment (rando2) [inline content/10.diagnostics.md:191]:
```suggestion
After 1 month, 98% of patients were seropositive for Spike IgG. 
```

File: content/10.diagnostics.md [hunk 5]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
+Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].
+
 <!--TODO: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->
-The duration over which these antibodies persist remains unknown.
+Evidence to date suggests that sustained immunity to SARS-CoV-2 occurs for a period of at least 6-8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @doi:10.1126/sciimmunol.abf8891].
+One study assessed sustained immunity using blood samples from 188 COVID-19 positive patients at various time points between 6 and 240 days post-symptom onset, whereby some patients were also assessed longitudinally [@doi:10.1126/science.abf4063]. 
+After 1 month, 98% of patients were seropositive for spike IgG. 
+Moreover, spike IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, whereby 90% of subjects were seropositive at 6 months.
```
Comment (rando2) [inline content/10.diagnostics.md:192]:
```suggestion
Moreover, Spike IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, with 90% of subjects seropositive at 6 months.
```
Is this saying that patients tended to follow their own, heterogeneous patterns over 6-8 months?  Also, it seems like most of their data was collected within 6 months, so we might want to be clear about the sample size for the 90%!

File: content/10.diagnostics.md [hunk 6]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
+Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.
+
```
Comment (rando2) [inline content/10.diagnostics.md:194]:
```suggestion
```

File: content/10.diagnostics.md [hunk 7]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
+Memory B cells and memory T cells were also measured.
+SARS-CoV-2 spike-specific memory B cell levels steadily rose over the first 120 days post-symptom onset.
```
Comment (rando2) [inline content/10.diagnostics.md:196]:
```suggestion
SARS-CoV-2 Spike-specific memory B cell levels steadily rose over the first 120 days following symptom onset. 
```

File: content/10.diagnostics.md [hunk 8]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
+One study assessed sustained immunity using blood samples from 188 COVID-19 positive patients at various time points between 6 and 240 days post-symptom onset, whereby some patients were also assessed longitudinally [@doi:10.1126/science.abf4063]. 
+After 1 month, 98% of patients were seropositive for spike IgG. 
+Moreover, spike IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, whereby 90% of subjects were seropositive at 6 months.
+Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.
+
+Memory B cells and memory T cells were also measured.
+SARS-CoV-2 spike-specific memory B cell levels steadily rose over the first 120 days post-symptom onset. 
+RBD-specific memory B cells were detected in COVID-19 patients 90 days post-symptom onset in previous studies [@doi:10.1016/j.cell.2020.11.029; @doi:10.1126/sciimmunol.abf8891], this study confirms these finding and shows that these cells steadily increase for over 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
```
Comment (rando2) [inline content/10.diagnostics.md:197]:
```suggestion
RBD-specific memory B cells had been detected in COVID-19 patients 90 days post-symptom onset in previous studies [@doi:10.1016/j.cell.2020.11.029; @doi:10.1126/sciimmunol.abf8891], and this study confirms these finding and shows that these cells steadily increase over the 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
```

File: content/10.diagnostics.md [hunk 9]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
+At 6 months, 92% of patients were positive for SARS-CoV-2 memory CD4+ T cells.
+Indeed, the spike-specific memory CD4+ T cells abundance over time were similar to the overall SARS-CoV-2-specific CD4+ T cells [@doi:10.1126/science.abf4063].
+T cell immunity to SARS-CoV-2 at 6-8 months post-symptom onset has also been confirmed by others [@doi:10.1101/2020.11.01.362319; @doi:10.1101/2020.12.08.416636].
+In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never before exposed to SARS-CoV-2. 
+This indicates that there is potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating coronaviruses that are responsible for the “common cold” [@doi:10.1016/j.cell.2020.05.015], but further research is required.
+There is concern that immunity may wane over time as there have been several reported cases of reinfection confirmed via genomic analysis of variants associated with infection in patients [@doi:10.1016/S1473-3099(20)30764-7; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
```
Comment (rando2) [inline content/10.diagnostics.md:207]:
```suggestion
There is concern that immunity may wane over time, as there several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient [@doi:10/ghfgkt; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
```
Does this work? I don't think we go into the evolution in this paper, so trying to just make a nod to how they were able to tell!

File: content/10.diagnostics.md [hunk 10]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
+Further research is required to determine the full extent of what level of sustained immunity can be achieved following SARS-CoV-2 infection
```
Comment (rando2) [inline content/10.diagnostics.md:208]:
```suggestion
Further research is required to determine the full extent to which sustained immunity can be and is typically achieved following SARS-CoV-2 infection
```

File: content/10.diagnostics.md [hunk 11]
```diff
@@ -180,11 +180,33 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
+Furthermore, it is unclear what repercussions there may be in terms of disease severity for patients [@doi:10.1016/S1473-3099(20)30783-0] and what reinfection implies for vaccine development and long-term efficacy [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481].
```
Comment (rando2) [inline content/10.diagnostics.md:209]:
```suggestion
Furthermore, it is unclear whether previous infection may carry repercussions in terms of disease severity for patients [@doi:10.1016/S1473-3099(20)30783-0] and what implications the possibility of reinfection holds for vaccine development and the long-term efficacy of vaccines [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481]. 
```

**Review by rando2 — COMMENTED** [review]
Thank you SO much for adding this, @RLordan! This is excellent.
I am wondering if we should include a brief sentence differentiating the implications of IgG vs B cells vs T cells for immunity over time? My immunology is weak, so I'm hesitant to take this on 😓 @nilswellhausen may be able to help if you want backup!

File: content/10.diagnostics.md (hunk 9 above)
Comment (RLordan) [reply content/10.diagnostics.md:208]: There is concern that immunity may wane over time, as several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient 

Delete "there"before several I think

What they mean by the genomic analysis here is that there was slight differences between the SARS-CoV-2 variant that caused the patients first infection, versus the second infection. They aren't necessarily referring to a different lineage such as B.1.1.7 etc. They just mean that they are sure it was a different infection. Does that make sense?

**Comment by RLordan** [comment]
> Thank you SO much for adding this, @RLordan! This is excellent.
> I am wondering if we should include a brief sentence differentiating the implications of IgG vs B cells vs T cells for immunity over time? My immunology is weak, so I'm hesitant to take this on 😓 @nilswellhausen may be able to help if you want backup!

Is this still required? let me know and I can tackle it at the weekend

File: content/10.diagnostics.md [hunk 12]
```diff
@@ -180,11 +180,34 @@ These results suggest that immunity to SARS-CoV-1 is sustained for at least a ye
...
+In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never before exposed to SARS-CoV-2. 
+This indicates that there is potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating coronaviruses that are responsible for the “common cold” [@doi:10.1016/j.cell.2020.05.015], but further research is required.
+There is concern that immunity may wane over time, as there several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient [@doi:10/ghfgkt; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
+Further research is required to determine the full extent to which sustained immunity can be and is typically achieved following SARS-CoV-2 infection
+Furthermore, it is unclear whether previous infection may carry repercussions in terms of disease severity for patients [@doi:10.1016/S1473-3099(20)30783-0] and what implications the possibility of reinfection holds for vaccine development and the long-term efficacy of vaccines [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481].
```
Comment (agitter) [inline content/10.diagnostics.md:210]:
```suggestion
Furthermore, it is unclear whether previous infection may carry repercussions in terms of disease severity for patients [@doi:10/fscx] and what implications the possibility of reinfection holds for vaccine development and the long-term efficacy of vaccines [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481]. 
```

**Review by rando2 — APPROVED** [review]

**Comment by rando2** [comment]
@RLordan I'm going to merge this in! I may end up trying to add in a little immuno background when I work on this paper (probably next after pharmaceuticals), so I may ask you for some help then if that's OK!

**Comment by RLordan** [comment]
> @RLordan I'm going to merge this in! I may end up trying to add in a little immuno background when I work on this paper (probably next after pharmaceuticals), so I may ask you for some help then if that's OK!

Super @rando2 will definitely help out

**Pull request merged**